tensorrt_llm/_torch/pyexecutor/resource_manager.py

```diff
@@ -532,7 +532,7 @@ def __init__(
         tp_size = mapping.tp_size
 
         # derive mamba parameters for conv and ssm states
-        d_inner = d_model * expand
+        d_inner = head_dim * num_heads
         conv_dim = d_inner + 2 * n_groups * d_state
         nheads = d_inner // head_dim
```

Comment:
same here. No need to compute `nheads` if it is passed as input (`num_heads`)

Reply:
fixed